utils/website/graphile-export/index.md

```diff
@@ -4,73 +4,24 @@ sidebar_position: 1
 
 # Graphile Export
 
-Graphile Export enables you to export a GraphQL Schema (or other code) as executable JavaScript code. You must, however, write your code in a way that makes it exportable; we have ESLint plugins to make this less onerous.
+Graphile Export enables you to export a GraphQL Schema (or other code) as
+executable JavaScript code. 
 
-## How it works
+The key reason to export your schema is to move schema introspection of postgres
+from runtime to build time. This results in:
 
-The system works by converting values in memory into source code strings. One of
-the key things that's challenging to export is functions (and function-derived
-things such as classes). In JavaScript you can see the source code of a function
-by calling `.toString()` on it:
+- Faster startup time
+- Reduced thundering herd on the database in the event of an outage
+- Much faster cold starts for Lambda
+- And probably more
 
-```js
-> (function add(a, b) { return a + b }).toString()
-'function add(a, b) { return a + b }'
-```
+Previously, in Postgraphile 4, export took the form of encoding the Postgres
+schema to JSON. Postgraphile 5 takes this further and generates runtime code,
+further cutting startup times.
```

Comment:
This should not be part of the graphile-export documentation; add it to the PostGraphile pages if you feel it warrants detail (though I'd personally keep it to the V4 migration guide).

Reply:
Same with paragraph below